asv_benchmarks/asv.conf.json

```
@@ -0,0 +1,163 @@
+{
+    // The version of the config file format.  Do not change, unless
+    // you know what you are doing.
+    "version": 1,
+
+    // The name of the project being benchmarked
+    "project": "nilearn",
+
+    // The project's homepage
+    "project_url": "https://nilearn.github.io",
+
+    // The URL or local path of the source code repository for the
+    // project being benchmarked
+    "repo": "..",
+
+    // The Python project's subdirectory in your repo.  If missing or
+    // the empty string, the project is assumed to be located at the root
+    // of the repository.
+    // "repo_subdir": "",
+
+    // Customizable commands for building, installing, and
+    // uninstalling the project. See asv.conf.json documentation.
+    "install_command": ["python -mpip install {wheel_file}"],
```

```suggestion
    "install_command": ["python -m pip install {wheel_file}"],
```
isn't a space required here?

---

The default install command from asv wasn't working, so copied this from sklearn: https://github.com/scikit-learn/scikit-learn/blob/a867ed7cc6f86146ce2a790531dbbcfb9cafe318/asv_benchmarks/asv.conf.json#L18

But let me try adding the space.